frontend/src/utils/testRenderer.tsx

```diff
@@ -23,7 +23,9 @@ export const render = (
 
     const Wrapper: FC = ({ children }) => (
         <UIProviderContainer>
-            <SWRConfig value={{ provider: () => new Map() }}>
+            <SWRConfig
+                value={{ provider: () => new Map(), dedupingInterval: 0 }}
```

Comment:
Interesting. Out of curiosity, what's the dedupingInterval do?

Reply:
swr sometimes serves cached data, github rabbit hole in their issues, suggested that this would help, basically bypassing cache